videozen-frontend/src/components/AllPlaylists/CreatedPlaylists/CreatedPlaylists.jsx

```diff
@@ -0,0 +1,74 @@
+import { Link } from "react-router-dom";
+
+import { useData } from "../../../context/data-context";
+
+import "./CreatedPlaylists.css";
+
+import Video from "../../Video/Video";
+
+const CreatedPlaylistsVideos = (props) => {
+  return (
+    <div>
+      <h3 className="text-left">{props.playlistName}</h3>
+      <div className="card-row">
+        {props.videos.map((video) => (
+          <Link
+            to={`/createdplaylist/${props.playlistId}/${video.id}`}
+            state={video}
+            style={{ textDecoration: "none", color: "black" }}
+          >
+            <Video
+              id={video.id}
+              title={video.title}
+              image={video.image}
+              ownerImage={video.ownerImage}
```

Comment:
Pass the whole `video `object as prop
```suggestion
              video={video}
```